modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -102,10 +148,12 @@ export default class EarthEngineLayer extends CompositeLayer {
     } = this.props;
 
     return (
-      getTileUrl &&
+      eeObject &&
       new TileLayer(
         this.getSubLayerProps({
-          id: getTileUrl(0, 0, 0)
+          // id: getTileUrl(0, 0, 0)
```

Comment:
Cut?

Reply:
This is on the to do list today: finding a good layer identifier for both static tiles and an `ImageCollection`. I expect we'll still use `getTileUrl`, or similar for static tiles.